README.md

```diff
@@ -2,83 +2,74 @@
 
 React Native's Animated library reimplemented.
 
-It provides a more comprehensive, low level abstraction for the Animated library API to be built on top of and hence allow for much greater flexibility especially when it comes to gesture based interactions.
+- **Native Performance**: Declare your animations in JS, but have them run on the native thread! 🧙‍♂️
+- **Precise Animations**: The API affords new levels of precision and detailed control of your animations. 🕹
+- **Backwards Compatible**: Use the same Animated library API from React Native that you've been using. You don't _need_ to change anything. 👍
```

Comment:
It's not true. It's right in most cases but sometimes it's not.

Reply:
Yeah, you are right. I can see how the last "Backwards Compatible" wording could be misleading. What I meant more here was you can start using it right away, only refactoring to the specific Reanimated API as you go... I don't know how best to word it, but I think the backwards compatibility is something that is a key value of Reanimated and should be highlighted at the top (I know a couple users that probably wouldn't be using Reanimated if they couldn't start this way).

Was there something else in this section that you think should be changed? Do you think the "Native Performance" or "Precise Animations" text should be changed?

Reply:
```suggestion
- **(mostly) Backwards Compatible**: Use the same Animated API from React Native that you've been using. You generally don't _need_ to change anything to get get started. 👍
```

Reply:
👌